Utilities/sysinfo.h

```diff
@@ -39,11 +39,11 @@ namespace utils
 
 	bool has_xop();
 
-	inline bool transaction_enter()
+	FORCE_INLINE bool transaction_enter(uint* out = nullptr)
 	{
 		while (true)
 		{
-			const auto status = _xbegin();
+			const uint status = _xbegin();
```

Comment:
why uint instead of u32

Reply:
It's correct type of the return value, doesn't need casting.